tests/integration/cached-decorator-test.ts

```diff
@@ -0,0 +1,53 @@
+import { module, test } from 'qunit';
+import { tracked, cached } from '@glimmer/tracking';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, settled } from '@ember/test-helpers';
+import { TestContext } from 'ember-test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+type Context = TestContext & {
+  personA: any;
+  personB: any;
+};
+
+module('Integration | Decorators | @cached', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it works', async function (this: Context, assert) {
+    class Person {
+      @tracked firstName = '';
+      @tracked lastName = '';
```

Comment:
```suggestion
      @tracked firstName: string;
      @tracked lastNam: string;
```

As these are definitely defined via the constructor, we can drop the initializes. :)